lms/templates/courseware/progress.html

```diff
@@ -104,6 +105,49 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
         <div class="grade-detail-graph" id="grade-detail-graph" aria-hidden="true"></div>
       %endif
 
+      %if credit_course is not None:
+        <div class="credit-eligibility">
+            <div class="credit-eligibility-container">
+                <div class="eligibility-heading">
+                    <h2>${_("Requirements for Course Credit")}</h2>
+                </div>
+                %if credit_course['eligibility_status'] == 'not_eligible':
+                    <span class="eligibility_msg">${student.username}, ${_("you are no longer eligible for this course")}</span>
+                %elif credit_course['eligibility_status'] == 'eligible':
+                    <span class="eligibility_msg">${student.username}, ${_("you have met the requirements for credit in this course.")}
+                        <a href="#">${_("Go to your dashboard")}</a> ${_("to purchase course credit.")}
+                    </span>
+                %elif credit_course['eligibility_status'] == 'partial_eligible':
+                    <span>${student.username}, ${_("you have not yet met the requirements for credit")}</span>
+                %endif
+                <button class="credit-help"><i class="fa fa-question"></i></button><br>
+                <div class="requirement-container is-hidden">
+                    %for requirement in credit_course['requirements']:
+                        <div class="requirement">
+                            <div class="requirement-name">${_(requirement['name'])}</div>
+                            <div class="requirement-status">
+                                %if requirement['status']:
+                                    %if requirement['status'] == 'submitted':
+                                        <span class="requirement-submitted">${_("Verification Submitted")}</span>
```

Comment:
For "Verification Submitted", is the verification process still in progress? Or does "Verification Submitted" mean that the verification was successful? I ask because we have a "Verification Failed" status, but I don't see a "Verification Successful" status.